tox.ini

```diff
@@ -25,7 +24,6 @@ setenv=
   filestore_osds_container: REGISTRY_ADDRESS = 192.168.55.1:5000
   bluestore_osds_container: REGISTRY_ADDRESS = 192.168.35.1:5000
   centos7: IMAGE_DISTRO = centos/7
-  jewel: CEPH_STABLE_RELEASE = jewel
   luminous: CEPH_STABLE_RELEASE = luminous
   ceph_ansible: CEPH_ANSIBLE_BRANCH = master
   ceph_ansible2.2: CEPH_ANSIBLE_BRANCH = stable-2.2
```

Comment:
shouldn't we remove this since we removed it from `envlist` ?

Reply:
(same thing at L34)

Reply:
Yes. Fixed. Thanks.